binary/src/aot64.rs

```diff
@@ -0,0 +1,28 @@
+use bytes::Bytes;
+use ckb_vm::machine::{aot::AotCompilingMachine, asm::AsmMachine};
+use std::env;
+use std::fs::File;
+use std::io::Read;
+use std::process::exit;
+
+fn main() {
+    let args: Vec<String> = env::args().skip(1).collect();
+
+    let mut file = File::open(args[0].clone()).unwrap();
```

Comment:
`std::fs::read`

Reply:
Merging this first so we can do aot tests in the VM repo, will use a separate commit to tackle those